src/CoreWCF.Primitives/src/CoreWCF/Channels/BufferedMessageWriter.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace CoreWCF.Channels
 {
-    internal abstract class BufferedMessageWriter
+    public abstract class BufferedMessageWriter
     {
```

Comment:
I think you should be able to copy this to WebHttp as it looks like it's not used outside of the message encoder.

Reply:
I could do that but it would have the side effect of making InternalBufferManager public.

Reply:
This as been addressed.